README.md

```diff
@@ -1,57 +1,15 @@
-# Northcoders News API
+## Creating environment variables
 
-## Background
+In order to be able to run this project, you will need to create environment variables to be able to connect to the database.
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+In the main project folder, please create these two files:
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+.env.test
 
-## Kanban
+.env.development
 
-### Link to your Trello Board here: https://trello.com/b/7yiHe1nI
+The content of .env.test should be as follows:
+PGDATABASE="nc_news_test"
```

Comment:
you'll want to take the "" from around the database names as this will throw an error